open-api/rest-catalog-open-api.yaml

```diff
@@ -2033,6 +2537,68 @@ components:
         metadata-location:
           type: string
 
+    CreateViewRequest:
+      type: object
+      required:
+        - name
+        - schema
+        - view-version
+        - properties
+      properties:
+        name:
+          type: string
+        location:
+          type: string
+        schema:
+          $ref: '#/components/schemas/Schema'
+        view-version:
+          $ref: '#/components/schemas/ViewVersion'
+        properties:
+          type: object
+          additionalProperties:
+            type: string
+
+    LoadViewResult:
+      description: |
+        Result used when a view is successfully loaded.
+
+
+        The view metadata JSON is returned in the `metadata` field. The corresponding file location of view metadata is returned in the `metadata-location` field.
```

Comment:
Looks like this line is too long. Can you check it against the rest of the file?

Reply:
I've checked and it's actually shorter than the one for `LoadTableResult`, so I've just tried to align everything similar to how other result descriptions are structured